neural_compressor/torch/quantization/config.py

```diff
@@ -74,10 +75,7 @@ class RTNWeightQuantConfig(BaseConfig):
         "enable_mse_search",
         "group_dim",
         "return_int",
-        "double_quant_dtype",
-        "double_quant_bits",
-        "double_quant_sym",
-        "double_quant_group_size",
+        "double_quant_config",
```

Comment:
`double_quant_config` -> `double_quant_type` ( one of ['GGML_TYPE_Q4_K', 'BNB'])